src/StreamError.cc

```diff
@@ -23,7 +23,8 @@
 #include "StreamError.h"
 #ifdef _WIN32
 #include <windows.h>
-#endif
+#include <io.h>
```

Comment:
Non-Windows systems (at least Linux, vxWorks) have  `isatty()` in unistd.h
```
#else /* _WIN32 */
#include <unistd.h>
```

Reply:
Thanks - now added